code/go/0chain.net/smartcontract/stakepool/lock.go

```diff
@@ -87,6 +86,10 @@ func (sp *StakePool) LockPool(
 
 		dp.Balance = b
 		dp.StakedAt = txn.CreationDate
+
+		update := newDelegatePoolUpdate(newPoolId, providerId, providerType)
+		update.Updates["balance"] = dp.Balance
```

Comment:
also add stakedAt to the update

Reply:
DelegatePool does not have a StakedAt member?